test/fixtures/html-entrypoint/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>FastbootTest</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- EMBER_CLI_FASTBOOT_TITLE --><!-- EMBER_CLI_FASTBOOT_HEAD -->
+
+  </head>
+  <body>
+    <!-- EMBER_CLI_FASTBOOT_BODY -->
+
+    <fastboot-script src="assets/vendor.js"></fastboot-script>
+    <fastboot-script src="assets/fastboot-test.js"></fastboot-script>
```

Comment:
How does embroider add the script meta tags in the final built index.html? I don't see it as part of test fixtures. Am I missing anything?

Reply:
That is definitely beyond the scope of this repo, but if you are curious it happens in embroider/core [here](https://github.com/embroider-build/embroider/blob/213960bbd62dcd70e067bb27bcfd7b357da127d2/packages/core/src/app.ts#L371) and [here](https://github.com/embroider-build/embroider/blob/213960bbd62dcd70e067bb27bcfd7b357da127d2/packages/core/src/app.ts#L388).

I'm not trying to be evasive here, I think it's important that we prioritize **interface over implementation**. If fastboot actually needed to care about where the script tags come from, then it would mean this is a bad interface because it leaks too much implementation.

Reply:
Oh, I misunderstood your question. Sorry. I was thinking about `<fastboot-script>`, but you must have meant`<meta name="your-app/config/environment">`.

I will make sure I show that as I incorporate all the feedback into this PR.